includes/admin/class-wc-rest-payments-webhook-controller.php

```diff
@@ -103,6 +103,14 @@ public function handle_webhook( $request ) {
 				case 'account.updated':
 					$this->account->refresh_account_data();
 					break;
+				case 'wcpay.notification':
+					if ( version_compare( WC_VERSION, '4.4.0', '>=' ) ) {
```

Comment:
This makes me think I may have missed something in my PR, I'm [not doing a version check there](https://github.com/Automattic/woocommerce-payments/pull/1044/files#diff-1cf209a1776d0208a1a6125d0ab5d3958b8631909cea0bce44b22f922eb1bdbfR670).

I haven't tested it explicitly, but the version of WC Admin bundled with our lowest supported version of WooCommerce makes a [similar call](https://github.com/woocommerce/woocommerce/blob/4.0.0/packages/woocommerce-admin/src/Notes/WC_Admin_Notes.php#L63) to `WC_Data_Store::load`. Is there something we're expecting to break when running < 4.4.0?

I think the version check is fine, especially given the context of the messages, I'm just wondering if I should be more defensive in my PR as well?